proxy/core/src/main/resources/cloudstate-common.conf

```diff
@@ -47,16 +54,17 @@ akka {
   cluster {
     shutdown-after-unsuccessful-join-seed-nodes = 60s
 
+    // FIXME consider settin distributed-data.max-delta-elements = <N>
+    // FIXME consider setting distributed-data.delta-crdt.max-delta-size = <N>
+
     sharding.state-store-mode = ddata
 
     // Non-durable for now, since we can't get native-image to work with lmdb right now
     sharding.distributed-data.durable.keys = []
 
-    // fixme Of course, this is not ideal, but not much choice at the moment.
-    auto-down-unreachable-after = 30s
-
     sharding {
       rebalance-interval = 5s
+      passivate-idle-entity-after = off // FIXME put in a good value here
```

Comment:
And remove the built-in passivation using receive timeout. Looks like `cloudstate.proxy.passivation-timeout` is currently 30s. Could substitute that setting here.

Reply:
@pvlugter I suspect that we need to remove the receive timeout thing at the same time to not have competing timers?

Reply:
Yes, remove the receive timeout at the same time. Although, I can imagine that passivation timeouts is something that will want to be defined per entity type on the user side? Maybe this needs its own issue to be enhanced.

Reply:
@pvlugter Makes sense to make the change separately from this PR. (Hence the FIXME)

Reply:
Sure. Would be good to capture in an issue then, rather than just a FIXME.